layouts/_default/term.html

```diff
@@ -1,8 +1,8 @@
 {{ define "main" }}
-    <h3 class="taxonomy-type section-title">{{ .Type | singularize | humanize }}</h3>
```

Comment:
This will break the rest of taxonomies, for example "tags".

Ref: https://deploy-preview-49--hugo-theme-stack.netlify.app/categories/test/